examples/express/server.mjs

```diff
@@ -81,22 +74,10 @@ app.listen(port, () => {
   console.log(`Express server listening on port ${port}`);
 });
 
-function purgeRequireCache() {
-  // purge require cache on requests for "server side HMR" this won't let
-  // you have in-memory objects between requests in development,
-  // alternatively you can set up nodemon/pm2-dev to restart the server on
-  // file changes, but then you'll have to reconnect to databases/etc on each
-  // change. We prefer the DX of this, so we've included it for you by default
-  for (const key in import.meta.cache) {
-    if (key.startsWith(BUILD_DIR)) {
-      delete import.meta.cache[key];
-    }
-  }
-}
-
 async function getContext(req) {
   const session = await AppSession.init(req, [env.SESSION_SECRET]);
 
+  console.error('\x1b[45m%s\x1b[0m', `requestGroupId=${crypto.randomUUID()}`);
```

Comment:
You might want to remove this line or store the id value in a variable to pass the same value to the client